src/rastervision/samples/workflow-configs/segmentation/deeplab-test.json

```diff
@@ -0,0 +1,110 @@
+{
+    "train_scenes": [
+        {
+            "id": "2-10",
+            "raster_source": {
+                "geotiff_files": {
+                    "uris": [
+                        "/opt/data/top_potsdam_2_10_RGBIR.tif"
+                    ]
+                }
+            },
+            "ground_truth_label_store": {
+                "segmentation_raster_file": {
+                    "src": {
+                        "geotiff_files": {
+                            "uris": [
+                                "/opt/data/top_potsdam_2_10_label_georeferenced.tif"
+                            ]
+                        }
+                    },
+                    "raster_class_map": [
+                        {
+                            "source_class": "#ffff00",
+                            "raster_vision_class": 255
+                        }
+                    ]
+                }
+            }
+        }
+    ],
+    "test_scenes": [
+        {
+            "id": "2-11",
+            "raster_source": {
+                "geotiff_files": {
+                    "uris": [
+                        "/opt/data/top_potsdam_2_11_RGBIR.tif"
+                    ]
+                }
+            },
+            "ground_truth_label_store": {
+                "segmentation_raster_file": {
+                    "src": {
+                        "geotiff_files": {
+                            "uris": [
+                                "/opt/data/top_potsdam_2_11_label_georeferenced.tif"
+                            ]
+                        }
+                    },
+                    "raster_class_map": [
```

Comment:
Also, what do you think about switch from `geotiff_files` to `image_file` so your script doesn't need to be run, esp. for new users?